src/pages/Homepage/Homepage.vue

```diff
@@ -85,10 +92,31 @@ export default {
 			return client.query({
 				query: homePageQuery
 			}).then(() => {
-				return ContentfulPage();
-			}).then(resolvedImport => {
+				return Promise.all([client.query({
+					query: experimentAssignmentQuery,
+					variables: { id: 'new_home_layout' }
+				})]);
+			}).then(async result => {
 				// Call preFetch for the active homepage
-				const component = resolvedImport.default;
+				let homePageComponent;
+				const experiment = result[0].data?.experiment;
+
+				switch (experiment?.version) {
+					case 'a':
+						// Should fetch the regular home page
+						homePageComponent = await ContentfulPage();
+						break;
+					case 'b':
+						// Should prefetch the experimental home page
+						homePageComponent = await ContentfulPage();
+						break;
+					default:
+						// Should redirect to /cps/home
+						homePageComponent = await ContentfulPage();
+						break;
```

Comment:
Can we please add a version `c` check for the redirect to `/cps/home` and set the standard ui homepage as the default/fallback condition.